endpoints/openrtb2/auction.go

```diff
@@ -309,6 +309,13 @@ func (deps *endpointDeps) parseRequest(httpRequest *http.Request) (req *openrtb_
 
 	lmt.ModifyForIOS(req.BidRequest)
 
+	// Integration Type Handling
+	err = deps.setIntegrationType(req)
+	if err != nil {
+		errs = []error{err}
+		return
+	}
```

Comment:
Nitpick: I think it's preferable to call the function and check if for the error on the same line here
`if err = deps.setIntegrationType(req); err != nil {`